src/Directory.Build.props

```diff
@@ -1,3 +1,12 @@
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'OpenTelemetry.sln'))\build\Common.prod.props" />
+  <PropertyGroup>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="README.md">
```

Comment:
Would this introduce broken links if the README.md file points to other files?

Reply:
Think we could override it in the related project files, it seems ok for most packages